game/input_handlers.py

```diff
@@ -3,6 +3,7 @@
 from typing import Callable, Optional, Tuple
 
 import tcod
+import numpy as np
```

Comment:
isort wants these in alphabetical order.
```suggestion
import numpy as np
import tcod
```